airqo-platform/AirQo-frontend
# validation and alert messages in ANALYTICS

- What does this PR do?
This PR addresses the missing validation and alert messages in ANALYTICS
- What JIRA task is related to this PR?
The link to the JIRA board task is [here](https://airqoteam.atlassian.net/browse/NLYTCS-103?atlOrigin=eyJpIjoiYjA4ZWIzMjRiMjYyNDllYmJhZjNiM2VmYTM3Y2VlNzkiLCJwIjoiaiJ9).
- How do I test out this PR?
Git clone this repo `cd AirQo-frontend/analytics` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run local` to test out the server accordingly. 
Since you are running your local development machine, you will also want to run the corresponding `AirQo-api/src/auth-service` (master branch) by running `npm run dev`.
> For a start, check out the validation messages on the registration page
![image](https://user-images.githubusercontent.com/1590213/86565038-d724a700-bf6f-11ea-898f-37315124b061.png)

![image](https://user-images.githubusercontent.com/1590213/87121143-cb482600-c28a-11ea-9539-07f553926fef.png)
